mapping.json

```diff
@@ -1,38 +1,48 @@
 {
     "google-analytics.com": [
-        ["google-analytics.com/ga.js", "ga.js"],
-        ["google-analytics.com/analytics.js", "analytics.js"],
-        ["google-analytics.com/inpage_linkid.js", "inpage_linkid.js"],
-        ["google-analytics.com/plugins/ga/inpage_linkid.js", "inpage_linkid.js"],
-        ["google-analytics.com/cx/api.js", "api.js"],
-        ["google-analytics.com/gtm/js", "gtm.js"]
+        { "url": "google-analytics.com/ga.js", "surrogate": "ga.js" },
```

Comment:
+1, that's a better format, no idea why I had arrays here before, that's never a good idea

Reply:
Oh, let's add an optional key for click to load entity. E.g. `"clickToLoadEntity": "Facebook, Inc."`. That would help us when we come to implement Chrome MV3 Click to Load support.

Reply:
I updated it to `regexRule` (we deprecated the plaintext one).

Regarding  `clickToLoadEntity`, is this just for the mapping.json?  I'd rather handle Mv3 specific stuff in a separate TD and followup PR, since it seems like that'll be on my plate next anyway.

Reply:
> I updated it to regexRule...

Ace, thanks for renaming to `regexRule`.

> (we deprecated the plaintext one)

How come you deprecated plantext rules? (I don't feel too strongly, but I wonder why we wouldn't use plaintext matching where that sufficed.)

> Regarding clickToLoadEntity, is this just for the mapping.json? I'd rather handle Mv3 specific stuff in a separate TD and followup PR, since it seems like that'll be on my plate next anyway.

Yea, that would just be for `mapping.json`. OK, fair enough if you'd like to do that as a follow-up. It's just if you made that change now it would make my life slightly easier. Well, if we at least agree to the name now that's enough I suppose. Is the name `clickToLoadEntity` what we will definitely use?

Reply:
We discussed `clickToLoadEntity` and I don't think we'll need it; we'll have that discussion in the MV3 CTL TD.

Going to merge here, thanks all for your careful reviews and insights!